sql/poly.sql

```diff
@@ -593,3 +593,13 @@ SELECT npoints( spoly '{
   (1.5121581120647 , -1.93925472462553e-05),
   (1.51214841579108 , -1.93925472462553e-05)
 }');
+
+SELECT set_sphere_output( 'RAD' );
+
+SELECT  spoint( spoly '{(0,0),(1,0),(1,1)}', 1 );
+
+SELECT  spoint( spoly '{(0,0),(1,0),(1,1)}', 2 );
+
+SELECT  spoint( spoly '{(0,0),(1,0),(1,1)}', 3 );
+
+SELECT  spoly_as_array( spoly '{(0,0),(1,0),(1,1)}' );
```

Comment:
I hate to nitpick whitespace, but I recommend removing one of the two spaces after the SELECTs on lines 599, 601, 603, and 605.

Otherwise, it looks good to me! But if you're adding a function, shouldn't there be an upgrade script and version number bump? Or will that be done in a separate PR?

Reply:
Of course, but which version should we put? @vitcpp

Reply:
As we discussed before, the master branch is the development branch. Once the API is changed I propose to assign a new version 1.3.0. Upgrade script should be named as pg_sphere--1.2.3--1.3.0.sql.in. The same as I proposed in #22.

Reply:
Except PR #22 already bumps the version to 1.3.0 and adds an upgrade script. I propose merging PR #22 first and then git rebase this PR on the new master to add this functionality to the future 1.3.0.